packages/paste-website/src/pages/components/ai-chat-log/index.mdx

```diff
@@ -0,0 +1,340 @@
+
+## Examples
+
+### Basic Message
+
+A message must include the author and body. Any message text from a user or a bot must be contained within the `AIChatMessageBody` component. Due to lengthy AI responses, the chat layout is now top-down.
+
+#### Bot
+
+<LivePreview
+  scope={{
+    AIChatLog,
+    AIChatMessage,
+    AIChatMessageAuthor,
+    AIChatMessageBody,
+  }}
+  language="jsx"
+  noInline
+>
+  {basicBotMessage}
+</LivePreview>
+
+#### User
+
+<LivePreview
+  scope={{
+    AIChatLog,
+    AIChatMessage,
+    AIChatMessageAuthor,
+    AIChatMessageBody,
+  }}
+  language="jsx"
+  noInline
+>
+  {basicHumanMessage}
+</LivePreview>
+
+### Message Body Sizes
+
+The `AIChatMessageBody` component has two sizes, `size="default"` and `size="fullScreen"`. The fullScreen size is used where the ChatLog is displayed in the full width of the page where larger font size is needed.
+
+<LivePreview
+  scope={{
+    AIChatLog,
+    AIChatMessage,
+    AIChatMessageAuthor,
+    AIChatMessageBody,
+  }}
+  language="jsx"
+>
+  {`<AIChatLog>
+  <AIChatMessage variant="user">
+    <AIChatMessageAuthor aria-label="You said">Gibby Radki</AIChatMessageAuthor>
+    <AIChatMessageBody size="default">
+      I'm a message that should be displayed in compact elements
+    </AIChatMessageBody>
+  </AIChatMessage>
+  <AIChatMessage variant="user">
+    <AIChatMessageAuthor aria-label="You said">Gibby Radki</AIChatMessageAuthor>
+    <AIChatMessageBody size="fullScreen">
+      I'm a message that will be displayed in full screen width
+    </AIChatMessageBody>
+  </AIChatMessage>
+</AIChatLog>`}
+</LivePreview>
+
+### Message with Actions
+
+Message actions can be used to provide quick responses or actions to the user.
+
+`AIChatMessageActionGroup` should be a child of `AIChatMessage` so that the text and meta information are correctly grouped together for assistive technologies. `AIChatMessageActionCard` also needs a readable `aria-label` that summarizes what the meta information says.
+
+Each item within `AIChatMessageActionGroup` should be wrapped with `AIChatMessageActionCard`. It is recommended to use reset button variants for content within `AIChatMessageActionCard`.
+
+Actions can still be added in `AIChatMessageBody` which are returned from the AI response.
+
+#### Feedback in AIChatMessageActionCard
+
+<LivePreview
+  scope={{
+    AIChatLog,
+    AIChatMessage,
+    AIChatMessageAuthor,
+    AIChatMessageBody,
+    AIChatMessageActionCard,
+    AIChatMessageActionGroup,
+    Button,
+    ThumbsUpIcon,
+    ThumbsDownIcon,
+    CopyIcon,
+    RefreshIcon,
+  }}
+  language="jsx"
+  noInline
+>
+  {botWithFeedback}
+</LivePreview>
+
+#### Buttons in AIChatMessageBody from AI Response
+
+<LivePreview
+  scope={{
+    AIChatLog,
+    AIChatMessage,
+    AIChatMessageAuthor,
+    AIChatMessageBody,
+    Button,
+    ButtonGroup,
+  }}
+  language="jsx"
+  noInline
+>
+  {botWithBodyActions}
```

Comment:
issue(blocking): these buttons need some padding, can you also go down to just 3 buttons so we're not showing off the wrapping bug until we fix it 😸 

<img width="750" alt="image" src="https://github.com/twilio-labs/paste/assets/75342690/6762c111-ad9f-429a-aca4-3666b6dacf42">

Reply:
Absolutely!